pkg/api/plugin.go

```diff
@@ -77,7 +77,9 @@ type DeployFn func(context.Context, map[string]interface{}) error
 
 // PluginConfig is passed into NewPlugin
 type PluginConfig struct {
-	TestConfig TestConfig
+	SSHSrcAddrPrefix string
+	SSHDestPortRange string
```

Comment:
don't believe we need to make SSHDestPortRange configurable

Reply:
Anubhuti mentions it in an email. I will double check with her just to make sure.